packages/contracts/tasks/helpers/Deployment.ts

```diff
@@ -375,6 +375,40 @@ export class Deployment {
     return contract as unknown as T;
   }
 
+  /**
+   * Deploy proxy clone instance.
+   *
+   * @param args deploy proxy clone arguments
+   * @returns clone contract
+   */
+  async deployProxyClone<C extends BaseContract, Args extends unknown[] = unknown[]>({
```

Comment:
One question: why did we define `deployProxyClone` instead of using the one in `import { deployProxyClone } from "@excubiae/contracts";` ?

Reply:
It's easier to add changes inside maci repo instead of updating excubiae.